docs/source/getting-started/examples/histograms.ipynb

```diff
@@ -191,8 +191,8 @@
     "    def privatize(s, t=1e8):\n",
     "        return preprocess >> dp.m.then_laplace_threshold(scale=s, threshold=t)\n",
     "    \n",
-    "    s = dp.binary_search_param(lambda s: privatize(s=s), d_in, d_out)\n",
-    "    t = dp.binary_search_param(lambda t: privatize(s=s, t=t), d_in, d_out)\n",
+    "    s = dp.binary_search(lambda s: privatize(s=s).map(d_in)[0] <= d_out[0])\n",
+    "    t = dp.binary_search(lambda t: privatize(s=s, t=t).map(d_in)[1] <= d_out[1])\n",
```

Comment:
The new version seems to be at a slightly lower level, and is slightly longer. Would the previous way still work?

Reply:
No, the previous way will throw an error about the tuples being incomparable. When searching, you're likely to find situations where epsilon is greater but delta is smaller, or vice versa, resulting in a failure to check.